tests/samples/rename_git.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/added b/moved
+similarity index 85%
+rename from added
+rename to moved
+index a071991..4dbab21 100644
+--- a/added
++++ b/moved
```

Comment:
We could also detect the normal case where renaming is not explicit but the diff contains deletion of a file and addition of another file which contains the same data as the deleted file, but this is not really what that patch is about.